tests/Doctrine/Tests/ORM/Cache/DefaultRegionTest.php

```diff
@@ -52,6 +51,23 @@ public function testSharedRegion()
         $this->assertTrue($region2->contains($key));
     }
 
+    /**
+     * @return Cache|null
+     */
+    private function createRegionCache()
+    {
+        if (extension_loaded('apc') && false !== @apc_cache_info()) {
+            return new ApcCache();
+        }
+
+        if ( ! extension_loaded('apcu')) {
```

Comment:
Wondering if `@requires ext-apc` at class level would do the trick here (PHPUnit internals)

Also, why are we using APC in tests, rather than a file- or array-cache?